File: lab3/lab3 app/app/src/main/java/com/pjatk/bmi/model/Recipe.java
package com.pjatk.bmi.model;

public class Recipe {
    private String title;
    private String[] ingredients;
    private String[] instructions;

    public Recipe(String title, String[] ingredients, String[] instructions) {
        this.title = title;
        this.ingredients = ingredients;
        this.instructions = instructions;
    }

    public String getTitle() {
        return title;
    }

    public String[] getIngredients() {
        return ingredients;
    }

    public String[] getInstructions() {
        return instructions;
    }
}

File: lab4/lab4 app/app/src/androidTest/java/com/pjatk/bmi/model/Recipe.kt
package com.pjatk.bmi.model

class Recipe(private val title: String, private val ingredients: Array<String>, private val instructions: Array<String>) {
    fun getTitle(): String {
        return title
    }

    fun getIngredients(): Array<String> {
        return ingredients
    }

    fun getInstructions(): Array<String> {
        return instructions
    }
}
